jira/client/client_test.go

```diff
@@ -0,0 +1,55 @@
+package client
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/athenianco/metadata/jira/types"
+	"github.com/stretchr/testify/require"
+)
+
+const issue = "AAA-10"
+
+var emptyUser types.User
+
+func TestCorruptedCreds(t *testing.T) {
+	c, err := New()
+	require.NoError(t, err)
+
+	ctx := context.Background()
+
+	_, _, err = c.Issue.GetWithContext(context.Background(), issue, nil)
+	require.Error(t, err)
+
+	ctx = types.WithUserContext(ctx, types.User{
+		Name:  "Johnny",
+		Token: "Bravo",
+		URL:   "http://example.com",
+	})
+
+	_, _, err = c.Issue.GetWithContext(context.Background(), issue, nil)
+	require.Error(t, err)
+}
+
+func TestValidCreds(t *testing.T) {
+	c, err := New()
+	require.NoError(t, err)
+
+	user := getUser()
+	if user == emptyUser {
```

Comment:
```suggestion
	if user == (types.User{}) {
```
And remove the `emptyUser` global.

Reply:
Done.

Reply:
Done.